fourthHomeWork/src/io/codelex/oop/exercise4/Runner.java

```diff
@@ -0,0 +1,27 @@
+package io.codelex.oop.exercise4;
+
+public enum Runner {
+    BEGINNER(50, 120),
+    INTERMEDIATE(30, 50),
+    ADVANCED(1, 30);
+
+    int minStart;
+    int minEnd;
+
+    Runner(int minStart, int minEnd) {
+        this.minStart = minStart;
+        this.minEnd = minEnd;
+    }
+
+    static void getFitnessLevel(int time) {
+        if (time < ADVANCED.minEnd) {
+            System.out.println(ADVANCED);
+        }
+        if (time > BEGINNER.minStart) {
```

Comment:
tad jau šeit arī labāk `else if`
savadāk ja pirmais būs true tad šis tāpat tiks lieki pāŗbaudīts